LICENSE

```diff
@@ -200,3 +200,347 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```

Comment:
There should probably be a bit of text at the top of `LICENSE` that states that this work is distributed under either license at the recipient's option.

Reply:
Currently, the LICENSE file simply contains the license text(s); it does not describe how the software is licensed. Although I agree this would be a useful improvement.

Reply:
Could we split into two files, `LICENSE.apache2.0` and `LICENSE.gpl2`?